tests/manual-test-cases/Group21-Registries/21-1-Whitelist.robot

```diff
@@ -42,9 +42,11 @@ Setup Harbor
     Simple ESXi Setup
 
     # Install a Harbor server with HTTPS a Harbor server with HTTP
+    Run Keyword and Ignore Error  Remove File  ${HARBOR_VERSION}.ova
     Install Harbor To Test Server  protocol=https  name=harbor-https
     Set Environment Variable  HTTPS_HARBOR_IP  %{HARBOR-IP}
 
+    Run Keyword and Ignore Error  Remove File  ${HARBOR_VERSION}.ova
```

Comment:
don't do this, if it gets to this point then the first install succeeded and we know the file is good.

Reply:
make sense , I will fix that